src/Workspaces/Core/Portable/Log/FunctionId.cs

```diff
@@ -11,464 +11,464 @@ internal enum FunctionId
         // for our other scenarios.
         TestEvent_NotUsed = 1,
 
-        WorkCoordinator_DocumentWorker_Enqueue,
-        WorkCoordinator_ProcessProjectAsync,
-        WorkCoordinator_ProcessDocumentAsync,
-        WorkCoordinator_SemanticChange_Enqueue,
-        WorkCoordinator_SemanticChange_EnqueueFromMember,
-        WorkCoordinator_SemanticChange_EnqueueFromType,
-        WorkCoordinator_SemanticChange_FullProjects,
-        WorkCoordinator_Project_Enqueue,
-        WorkCoordinator_AsyncWorkItemQueue_LastItem,
-        WorkCoordinator_AsyncWorkItemQueue_FirstItem,
-
-        Diagnostics_SyntaxDiagnostic,
-        Diagnostics_SemanticDiagnostic,
-        Diagnostics_ProjectDiagnostic,
-        Diagnostics_DocumentReset,
-        Diagnostics_DocumentOpen,
-        Diagnostics_RemoveDocument,
-        Diagnostics_RemoveProject,
-        Diagnostics_DocumentClose,
+        WorkCoordinator_DocumentWorker_Enqueue = 2,
```

Comment:
what's the benefit here?

Reply:
That you don't accidentally change the values by deleting (commenting out) a member that's not longer used or adding one in the middle.